src/test/java/com.gigssandbox/ConsoleHelperTest.java

```diff
@@ -0,0 +1,87 @@
+package com.gigssandbox;
+
+import com.gigssandbox.entities.Band;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class ConsoleHelperTest {
+    private ByteArrayOutputStream output;
+    private PrintStream oldOut;
+    private ConsoleHelper helper;
+
+    @Before
+    public void setUp() {
+        System.setIn(new ByteArrayInputStream("1 lorem ipsum is not actually a fish".getBytes()));
+        oldOut = System.out;
+        output = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(output));
+        helper = new ConsoleHelper();
+    }
+
+    @After
+    public void cleanUpStreams() {
+        System.setIn(System.in);
+        System.setOut(oldOut);
+    }
+
+    @Test
+    public void shouldWriteStringToConsoleWhenMethodIsCalled() {
+        String testString = "lorem ipsum is not actually a fish";
+        helper.writeStringToConsole(testString);
+        Assert.assertEquals(testString, output.toString().replaceAll("[\n\r]", ""));
```

Comment:
Please try to split test body into three parts (given, when, then) using empty lines to be able understand author's intention more clearly:

```
    @Test
    public void shouldWriteStringToConsoleWhenMethodIsCalled() {
        String testString = "lorem ipsum is not actually a fish"; 

        helper.writeStringToConsole(testString);  

        Assert.assertEquals(testString, output.toString().replaceAll("[\n\r]", "")); 
    }
```